### ISSUE TITLE
Clarify relationship between 3ID and IDX

### ISSUE THREAD

**Issue by stbrody** [3box/docs#20]
Neither https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-79/CIP-79.md nor https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-20/CIP-20.md make any mention of "IDX", but according to @oed there are core parts of the 3id that are stored in IDX, and it isn't possible to resolve a 3id DID document without using IDX.

**Comment by stbrody** [comment]
From [CIP-20](https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-20/CIP-20.md): "Adding a new key pair to the 3ID Keychain does not disclose a public link between that account and the DID"

Is that actually true?  If the link is stored in IDX and/or a CAIP-10 link then there is indeed a publicly disclosed link between the authentication blockchain account and the DID.

**Comment by stbrody** [comment]
Also from CIP-20: "The 3ID Keychain is a definition that contains".  I guess this means an IDX definition, but that isn't stated at all

**Comment by oed** [comment]
> Is that actually true? If the link is stored in IDX and/or a CAIP-10 link then there is indeed a publicly disclosed link between the authentication blockchain account and the DID.

Yes this is true. We just use the entropy of a signature to generate a deterministic `did:key` which is publicly linked to the keychain and 3ID.
